### DISCUSSION TITLE
Format node for code

### DISCUSSION THREAD

**Discussion by fgionghi** [freeplane/freeplane#2044] [Help]
Hi, hoping is the right place to ask.
I just would like to paste some snippet code sometimes inside my maps.
There is any way I can format it decently?
Thank you

**Comment by euu2021** [comment]
I just paste inside the text node and it works fine. You can also use Markdown to have code blocks.

If you are as asking about syntax highlight, see https://github.com/freeplane/freeplane/issues/1423

This may also be relevant: https://github.com/freeplane/freeplane/issues/408

**Comment by dpolivaev** [comment]
If your IDE or browser gives you HTML formatted code snippets, copy and paste might be an easy way to have syntax highlighting in your snippets.

**Reply by euu2021** [reply]
Nice. I did not think about it before.

**Comment by lilive** [comment]
Hello,
I've made these scripts : https://github.com/lilive/Freeplane-Scripts/tree/master/Format/FormatCode
